Makefile

```diff
@@ -157,7 +157,7 @@ DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace $(
 ###############################################################################
 ###                                Protobuf                                 ###
 ###############################################################################
-PROTO_BUILDER_IMAGE=ghcr.io/cosmos/proto-builder:0.9.0
+PROTO_BUILDER_IMAGE=ghcr.io/cosmos/proto-builder:0.14.0
```

Comment:
seems like the recommended version was 0.11.2 is there any difference with this version?

Reply:
Currently recommend version is > 0.11.5 for sdk 47.
Using 0.14.0 just to use the most upto date builder.
The latest image has some optimizations as well https://github.com/cosmos/cosmos-sdk/pull/16343